integreat_cms/core/settings.py

```diff
@@ -611,7 +611,8 @@
 REQUEST_DEBUG_USERS = [
     item.strip()
     for item in os.environ.get("INTEGREAT_CMS_REQUEST_DEBUG_USERS", "").split(",")
-].remove("")
+    if item.strip() != ""
```

Comment:
Why the extra comparison with an empty string? This should be [more pythonic](https://docs.python.org/3/library/stdtypes.html#truth-value-testing):
```suggestion
    if item.strip()
```